{"name": "Test case ID: TS45", "status": "broken", "statusDetails": {"message": "AttributeError: 'str' object has no attribute 'to_dict'", "trace": "test_setup = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"587e7dd007defd9e3c97bd8af110dea2\")>\n\n    @allure.description(\"navigate to twitter\")\n    @allure.severity(severity_level=\"MEDIUM\")\n    @allure.title(\"Test case ID: TS45\")\n    def test_3_navigate_to_twitter(test_setup):\n        try:\n            driver = test_setup\n            hp = HomePage(driver)\n            fo = Footer(driver)\n            hp.click_save_button()\n            fo.navigate_to_instagram()\n>           assert driver.current_url == 'https://www.twitter.com/'\nE           AssertionError: assert 'https://www.instagram.com/' == 'https://www.twitter.com/'\nE             - https://www.twitter.com/\nE             ?              ^^^^^\nE             + https://www.instagram.com/\nE             ?             +++ ^^ ++\n\ntests\\test_footer.py:52: AssertionError\n\nDuring handling of the above exception, another exception occurred:\n\ntest_setup = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"587e7dd007defd9e3c97bd8af110dea2\")>\n\n    @allure.description(\"navigate to twitter\")\n    @allure.severity(severity_level=\"MEDIUM\")\n    @allure.title(\"Test case ID: TS45\")\n    def test_3_navigate_to_twitter(test_setup):\n        try:\n            driver = test_setup\n            hp = HomePage(driver)\n            fo = Footer(driver)\n            hp.click_save_button()\n            fo.navigate_to_instagram()\n            assert driver.current_url == 'https://www.twitter.com/'\n        except Exception as e:\n            print(e)\n>           driver.print_page(\"test_TS45_error.png\")\n\ntests\\test_footer.py:55: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"587e7dd007defd9e3c97bd8af110dea2\")>, print_options = 'test_TS45_error.png'\n\n    def print_page(self, print_options: Optional[PrintOptions] = None) -> str:\n        \"\"\"Takes PDF of the current page.\n    \n        The driver makes a best effort to return a PDF based on the\n        provided parameters.\n        \"\"\"\n        options = {}\n        if print_options:\n>           options = print_options.to_dict()\nE           AttributeError: 'str' object has no attribute 'to_dict'\n\nvenv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py:610: AttributeError"}, "description": "navigate to twitter", "attachments": [{"name": "stdout", "source": "63b80d63-71af-41a7-a0ee-c8d0beb7d112-attachment.txt", "type": "text/plain"}], "start": 1680263468882, "stop": 1680263471564, "uuid": "f0a401bc-ae19-4cc6-b742-2c6a4d0d33a1", "historyId": "2634faee7537e212d54da32399ff4092", "testCaseId": "d264974fcb668296be286174654546cd", "fullName": "tests.test_footer#test_3_navigate_to_twitter", "labels": [{"name": "severity", "value": "MEDIUM"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_footer"}, {"name": "host", "value": "DESKTOP-9HU6DH7"}, {"name": "thread", "value": "24172-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_footer"}]}